build/build_test.go

```diff
@@ -6,12 +6,23 @@ import (
 	"testing"
 )
 
+/*
+// TestBuild function
 func TestBuild(t *testing.T) {
 	got := build.Runfile()
```

Comment:
It's a matter of preference, but I like to define my 'want' before I get my 'got'. It's like saying "Okay, here's what I'm _expecting_... now let's see what we _get_!"

Reply:
Noted and updated